macOS/DuckDuckGo/RemoteMessaging/ActiveRemoteMessageModel.swift

```diff
@@ -68,10 +76,12 @@ final class ActiveRemoteMessageModel: ObservableObject {
     init(
         remoteMessagingStore: @escaping @autoclosure () -> RemoteMessagingStoring?,
         remoteMessagingAvailabilityProvider: RemoteMessagingAvailabilityProviding?,
-        openURLHandler: @escaping (URL) async -> Void
+        openURLHandler: @escaping (URL) async -> Void,
+        navigateToFeedbackHandler: @escaping () async -> Void
```

Comment:
This could be a protocol type similar to `NewTabPageLinkOpening` - it defines a function that takes an enum describing the navigation target. Perhaps it could also incorporate the `openURLHandler` functionality.

But since we have a single navigation type for now and we don’t have much time, I’m fine with proceeding with it as is.